### ISSUE TITLE
Gather benchmarks

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#2]
Gather benchmarks of first-rate movement, member, network org websites in a [slide deck](https://docs.google.com/presentation/d/11jndrd2fzjO_1LPdbZUwQOYys6WxspqwInmz0gjJYvY/edit).

**Comment by prefontaine** [comment]
I'll go poke around and get back to you.

**Comment by prefontaine** [comment]
I decided to look for good "people" pages. I figured this made sense
because we're essentially highlighting a group of people that others are
going to decide whether or not to engage with — spending time rather than
money or hiring them. Still, these are folks you're gonna decide to "work"
with. Really just first blush from a quick search, looking for what was
compelling and had nice interactions. See notes for each.
https://www.pinterest.com/prefontaine/people-inspirations/
